src/util/irep.h

```diff
@@ -15,6 +15,7 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <vector>
 
 #include "deprecate.h"
+#include "invariant.h"
```

Comment:
Please remove the `cassert` header now that this move is taking place.

Reply:
Ah yes. Done. Also in other PRs.